enhancements/installer/vsphere-ipi-zonal-external-lb.md

```diff
@@ -0,0 +1,201 @@
+---
+title: vsphere-ipi-zonal-external-lb
+authors:
+  - "@jcpowermac"
+reviewers:
+  - "@rvanderp3"
+  - "@bostrt"
```

Comment:
we need assisted installer team to chime in since it will affect vsphere integration of AI.
@oourfali @filanov @gamli75